fastlane/Fastfile

```diff
@@ -76,6 +79,9 @@ end
 
     ios_bump_version_release()
     new_version = ios_get_app_version()
+    extract_release_notes_for_version(version: new_version,
+      release_notes_file_path: File.join(ENV["PROJECT_ROOT_FOLDER"], 'RELEASE-NOTES.txt'),
+      extracted_notes_file_path: File.join(PROJECT_ROOT_FOLDER, 'Simplenote', 'Resources', 'release_notes.txt'))
```

Comment:
💄 nit: We realized a bit too late in the other PRs that we were not consistent here, and should use `PROJECT_ROOT_FOLDER` everywhere instead of the `ENV` var 🤷  No big deal at all, but figured while we were at it…

```suggestion
      release_notes_file_path: File.join(PROJECT_ROOT_FOLDER, 'RELEASE-NOTES.txt'),
      extracted_notes_file_path: File.join(PROJECT_ROOT_FOLDER, 'Simplenote', 'Resources', 'release_notes.txt'))
```